packages/xdl/src/Simulator.js

```diff
@@ -153,9 +153,37 @@ export async function _isSimulatorRunningAsync() {
   }
 
   let bootedDevice = await _bootedSimulatorDeviceAsync();
+  
+  if (!bootedDevice) {
+    return await _bootDefaultSimulatorDeviceAsync();
+  }
+  
   return !!bootedDevice;
 }
 
+async function _bootDefaultSimulatorDeviceAsync() {
+  Logger.global.info(`Booting device in iOS simulator...`);
+  try {
+    let defaultDeviceUDID = await _getDefaultSimulatorDeviceUDIDAsync();
+    if (!defaultDeviceUDID) {
+      defaultDeviceUDID = (await _getFirstAvailableDeviceAsync()).udid;
+    }
+    return await _xcrunAsync(['simctl', 'boot', defaultDeviceUDID]);
+  } catch (e) {
+    Logger.global.error(`There was a problem booting a device in iOS Simulator. Quit Simulator, and try again.`);
+    throw e;
+  }
+}
+
+async function _getDefaultSimulatorDeviceUDIDAsync() {
+  try {
+    const { stdout: defaultDeviceUDID } = await spawnAsync('defaults', ['read', 'com.apple.iphonesimulator', 'CurrentDeviceUDID']);
+    return defaultDeviceUDID.replace(/\n$/, '');
+  } catch (e) {
+    return false;
```

Comment:
```suggestion
    return null;
```

Reply:
This is to make the type of this function simpler (nullable `string` vs `string | boolean`).